typescript/cli/src/context/strategies/chain/MultiChainResolver.ts

```diff
@@ -134,26 +134,24 @@ export class MultiChainResolver implements ChainResolver {
       chains.push(argv.origin);
     }
 
-    if (argv.destination) {
-      chains.push(argv.destination);
-    }
-
     if (argv.chain) {
       chains.push(argv.chain);
     }
 
-    if (!argv.chains && chains.length === 0) {
+    if (argv.chains) {
```

Comment:
does this mean that if `destination` is defined we don't want to include it in here? also wondering what happens if someone passes `", "` as `chains` input and `additionalChains` ends up empty

Reply:
yes - we're giving precedence to `origin` and `chain` and `chains` before finally looking at `destination`